functions/src/send-email-to-manager.ts

```diff
@@ -24,26 +25,23 @@ export const sendEmailToManager = functions.firestore
 
     const vote = voteSnapshot.data()! as Vote;
 
-    const latestImport = await firebase
-      .firestore()
-      .collection("employee-imports")
-      .orderBy("at", "desc")
-      .limit(1)
-      .get()
-      .then(result => result.docs[0]);
-    if (!latestImport) {
-      throw new Error("cannot find latest employee data import");
-    }
-
-    const employeeSnapshot = await latestImport.ref
+    const employeeSnapshot = await db
       .collection("employees")
-      .doc(vote.voter)
+      .doc(vote.email)
       .get();
+    if (!employeeSnapshot) {
+      throw new Error("cannot find employee data import");
+    }
     const employee = employeeSnapshot.data();
     if (!employee) {
-      throw new Error(`cannot find user '${vote.voter}' in employee data`);
+      throw new Error(`cannot find user '${vote.email}' in employee data`);
     }
     if (!employee.managerEmail) {
+      console.error(
+        `Employee ${
+          employee.fullName
+        }'s manager isn't recorded into the database yet`
+      );
```

Comment:
This can be normal. There is at least one person that has no manager: the CEO. I suggest using the info log level and using the `aborting` word like in other places: 

```
console.info(`employee '${employee.fullName}' has no manager; aborting`)
```